src/BlazorUI/Bit.BlazorUI/Components/Surfaces/Callout/BitCallout.scss

```
@@ -14,28 +15,312 @@
+    pointer-events: none;
+}
+
 .bit-clo-ovl {
     top: 0;
     left: 0;
     width: 100%;
     height: 100%;
     display: none;
     position: fixed;
-    z-index: $zindex-overlay;
+    // A callout opened from inside another one is stacked above it, and so is the overlay that takes the
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)